byteparse/shared/src/test/scala/byteparse/classparse/ClassTests.scala

```diff
@@ -0,0 +1,1482 @@
+          BasicElemPool(StringElem("Some other month")),
+          BasicElemPool(StringElem("Switch Case Result: ")),
+          BasicElemPool(StringElem("maybe")),
+          NameAndType("hashCode", "()I"),
+          BasicElemPool(StringElem("yes")),
+          NameAndType("equals", "(Ljava/lang/Object;)Z"),
+          BasicElemPool(StringElem("no")),
+          BasicElemPool(StringElem("You answered yes.")),
+          BasicElemPool(StringElem("You answered no.")),
+          BasicElemPool(StringElem("You answered maybe.")),
+          BasicElemPool(StringElem("You answered ")),
+          BasicElemPool(StringElem("A")),
+          BasicElemPool(StringElem("B")),
+          BasicElemPool(StringElem("123")),
+          Class("java/lang/Integer"),
+          NameAndType("parseInt", "(Ljava/lang/String;)I"),
+          NameAndType("toString", "(I)Ljava/lang/String;"),
+          BasicElemPool(StringElem("CodeTest")),
+          BasicElemPool(StringElem("[I")),
+          BasicElemPool(StringElem("[Ljava/lang/String;")),
+          BasicElemPool(StringElem("[Z")),
+          BasicElemPool(StringElem("java/lang/System")),
+          BasicElemPool(StringElem("out")),
+          BasicElemPool(StringElem("Ljava/io/PrintStream;")),
+          BasicElemPool(StringElem("java/io/PrintStream")),
+          BasicElemPool(StringElem("println")),
+          BasicElemPool(StringElem("(Ljava/lang/String;)V")),
+          BasicElemPool(StringElem("print")),
+          BasicElemPool(StringElem("java/lang/Double")),
+          BasicElemPool(StringElem("valueOf")),
+          BasicElemPool(StringElem("(D)Ljava/lang/Double;")),
+          BasicElemPool(StringElem("printf")),
+          BasicElemPool(StringElem("(Ljava/lang/String;[Ljava/lang/Object;)Ljava/io/PrintStream;")),
+          BasicElemPool(StringElem("(Ljava/math/BigInteger;I)V")),
+          BasicElemPool(StringElem("append")),
+          BasicElemPool(StringElem("(Ljava/lang/String;)Ljava/lang/StringBuilder;")),
+          BasicElemPool(StringElem("(I)Ljava/lang/StringBuilder;")),
+          BasicElemPool(StringElem("toString")),
+          BasicElemPool(StringElem("()Ljava/lang/String;")),
+          BasicElemPool(StringElem("(D)Ljava/lang/StringBuilder;")),
+          BasicElemPool(StringElem("(I)V")),
+          BasicElemPool(StringElem("hashCode")),
+          BasicElemPool(StringElem("()I")),
+          BasicElemPool(StringElem("equals")),
+          BasicElemPool(StringElem("(Ljava/lang/Object;)Z")),
+          BasicElemPool(StringElem("java/lang/Integer")),
+          BasicElemPool(StringElem("parseInt")),
+          BasicElemPool(StringElem("(Ljava/lang/String;)I")),
+          BasicElemPool(StringElem("(I)Ljava/lang/String;"))
+        ))
+
+        assert(parsedClass.methods(1).attributes(0).asInstanceOf[CodeAttribute].code ==
+            ArrayBuffer(
+              GetStatic(2),          //   0: getstatic     #2                  // Field java/lang/System.out:Ljava/io/PrintStream;
```

Comment:
Could we add a comment saying that the comments on the right are coming from `javap`? Otherwise it's not clear whether they're the output of a third-party parser or whether they're the output of the byteparse parser reformatted into a third-party-parser's syntax